Updated peer production/community aspect of the game + changed prompts

ca-lida opened the pull request:
Updated readme to reflect the actual peer production/community focus of the game. Used taguette highlights for prompts + some chatgpt + personal thoughts to generate new prompts and responses.

Student can add to the prompt and response csv files with their ideas to improve on the issues discussed in class.

I also suggest changing the name of the repo, this is not about LLMs.

ca-lida committed:
Updated README + prompt and response csv

Updated readme to reflect the actual peer community focus of the game. Used taguette highlights for prompts + some chatgpt + personal thoughts to generate new prompts and responses.

mdbesten committed:
Create label.yml

Matthieu-z-David commented:
Nice contribution, I agree on the renaming of the overall project. I saw other contribution regarding the prompts that might interest you ;)

EduardoSMarques committed:
Merge branch 'mdbesten:master' into master

ca-lida committed:
Deleted unrelated PCIO

we can convert to PCIO from the csv's later

ca-lida committed:
Merge branch 'master' of https://github.com/ca-lida/what-could-go-wrong-peer-production

B4dCow committed:
Update prompts.csv

Add some prompts about peer production

Matthieu-z-David committed:
Update prompts.csv

Adding some prompts regarding IP

B4dCow committed:
Update prompts.csv

Fixing non utf 8 character

B4dCow committed:
Update prompts.csv

delete non utf8 character

Matthieu-z-David committed:
Update prompts.csv

Undo what was done

B4dCow committed:
Merge branch 'master' into new-prompts-branch

B4dCow committed:
Merge pull request #8 from ca-lida/new-prompts-branch

New prompts and delete non0-utf8 character

B4dCow committed:
Balance in the Card Czard Distribution

mdbesten committed:
Create labeler.yml

Matthieu-z-David committed:
Update responses.csv

Adding responses building on @B4dCow prompts (Nicolas)

mdbesten committed:
Update labeler.yml

quotes around regex

Matthieu-z-David committed:
Merge pull request #12 from ca-lida/MatthieuContribution

Update responses.csv

Thank you for the approval

Naopod committed:
Merge branch 'mdbesten:iren' into Anthony's-branch

Naopod committed:
Added time limit rules

YurenJeremyJin committed:
Launch the anonymous poll after discussion during the game

Naopod committed:
Added time limit rules

YurenJeremyJin committed:
Launch an anonymous poll after discussion

yunjieee committed:
Update responses.csv

I updated responses words. I hope this modification can enlarge the fun of the game. Have fun in this game!

vallathortense committed:
gram_test

vallathortense committed:
Merge pull request #19 from ca-lida/hortense

grammar correction

Naopod committed:
Prompts leading discussion

stellanweadau committed:
Merge pull request #11 from ca-lida/new-prompts-branch

Balance in the Card Czard Distribution

Naopod committed:
prompts leading discussion

Naopod committed:
PROMPTS for leading discussion

yunjieee committed:
Update README.md

stellanweadau committed:
change titles for more clarity

yunjieee committed:
Update responses.csv

yunjieee committed:
Update responses.csv

yunjieee committed:
Update prompts.csv

Naopod committed:
Merge branch 'master' into Anthony's-branch

EduardoSMarques committed:
Update main.tex

- Challenge card

Matthieu-z-David committed:
Merge branch 'master' into Anthony's-branch

Matthieu-z-David committed:
Merge pull request #16 from ca-lida/Anthony's-branch

Anthony's branch

Naopod committed:
Merge branch 'master' into Yunjie-branch

EduardoSMarques committed:
Create prompts_pt.csv

- prompts in portuguese

EduardoSMarques committed:
Update prompts_pt.csv

- added head

EduardoSMarques committed:
Create responses_pt.csv

- reponses in portuguese

Matthieu-z-David committed:
Merge pull request #22 from ca-lida/Eduardo

Portuguese language + Challenge card

EduardoSMarques committed:
Update README.md

- added challenge card to the gameplay

EduardoSMarques committed:
Create challenges.csv

- List of challenges

EduardoSMarques committed:
Create challenges_pt.csv

- Challenges in Portuguese

EduardoSMarques committed:
Merge branch 'master' into Eduardo

EduardoSMarques committed:
Merge pull request #24 from ca-lida/Eduardo

Game Play update + Challenges

stellanweadau committed:
Merge pull request #18 from ca-lida/Yunjie-branch

Update responses.csv

ca-lida committed:
Updated Game Set Up instructions.

Includes instructions for csv, not pcio.

stellanweadau committed:
additional point for the discussion card owner

B4dCow committed:
Merge pull request #25 from ca-lida/stellan-work

YurenJeremyJin committed:
Merge branch 'master' into Yuren-Branch

yunjieee committed:
Merge pull request #17 from ca-lida/Yuren-Branch

 Launch an anonymous poll after discussion part
